pkg/kubectl/stop.go

```diff
@@ -337,6 +333,8 @@ func (reaper *StatefulSetReaper) Stop(namespace, name string, timeout time.Durat
 	}
 	if timeout == 0 {
 		numReplicas := ss.Spec.Replicas
+
+		// BUG: this timeout is never used.
```

Comment:
That might not be a bug though. It's implementing the `type Reaper interface`, so it needs to take the `timeout` parameter, whether it needs it or not.